src/main.js

```diff
@@ -18,54 +19,18 @@ function createMainWindow() {
     },
   });
 
-  mainWindow.loadFile("index.html");
+  mainWindow.loadURL(`file://${__dirname}/index.html`);
```

Comment:
Had to change the way index.html was loaded due to an exception being thrown after building the application